spec/features/work_packages/details/inplace_editor/subject_editor_spec.rb

```diff
@@ -11,6 +11,7 @@
   let(:field) { WorkPackageField.new page, property_name }
 
   before do
+    maximize(page)
     allow(User).to receive(:current).and_return(user)
     visit project_work_packages_path(project)
```

Comment:
You can explicitly use [shared context](https://www.relishapp.com/rspec/rspec-core/docs/example-groups/shared-context) here. And it could also include going the page as this is the same for all the  mentioned spec files.